src/preferences/listener.test.ts

```diff
@@ -85,25 +78,26 @@ describe("preferences/listener", function () {
     assert.ok(updatePreferencesStub.notCalled);
   });
 
-  for (const [previewSetting, previewContextValue] of [
-    [ENABLE_PRODUCE_MESSAGES, contextValues.ContextValues.produceMessagesEnabled],
-  ]) {
-    for (const enabled of [true, false]) {
-      it(`should update the "${previewContextValue}" context value when the "${previewSetting}" setting is changed to ${enabled} (REMOVE ONCE PREVIEW SETTING IS NO LONGER USED)`, async () => {
-        getConfigurationStub.returns({
-          get: sandbox.stub().withArgs(previewSetting).returns(enabled),
-        });
-        const mockEvent = {
-          affectsConfiguration: (config: string) => config === previewSetting,
-        } as ConfigurationChangeEvent;
-        onDidChangeConfigurationStub.yields(mockEvent);
-
-        createConfigChangeListener();
-        // simulate the setting being changed by the user
-        await onDidChangeConfigurationStub.firstCall.args[0](mockEvent);
-
-        assert.ok(setContextValueStub.calledWith(previewContextValue, enabled));
-      });
-    }
-  }
+  // TODO: uncomment this when we have preview/experimental settings again
+  // for (const [previewSetting, previewContextValue] of [
+  //   // [SETTING_CONST, contextValues.ContextValues.valueHere],
+  // ]) {
```

Comment:
Planning to revisit this once https://github.com/confluentinc/vscode/pull/513 is updated with an experimental setting